releasenotes/notes/dca_affinity-0326838837e21d5e.yaml

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
Such release notes are mandatory in the [Datadog/datadog-agent](https://github.com/DataDog/datadog-agent/tree/main/releasenotes/notes) repository.
For this [Datadog/datadog-operator](https://github.com/DataDog/datadog-operator) PR, we can remove it.
The [release notes will be automatically generated from the git history](https://github.com/DataDog/datadog-operator/releases).